index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+$nazevParty = "Czechitas";
+$poradatel = "Ondra, Petr, Jirka";
+$misto = "Praha";
+$kapacita = 30;
+
+$ucastnici = explode("\n", file_get_contents("ucastnici.txt"));
+
+if (isset($_GET["pridej"])) {
+    $novyUcastnik = $_GET["pridej"];
+    $ucastnici[] = $novyUcastnik;
+
+    file_put_contents("ucastnici.txt", implode("\n", $ucastnici));
+
+    header("Location: /");
+    exit;
+}
+
+$pocetUcastniku = count($ucastnici);
+
+$volnaKapacita = $kapacita - $pocetUcastniku;
+
+if ($pocetUcastniku == 0) {
+    $pocetUcastniku = "Nikdo nepřijde :-(";
+}
+if ($volnaKapacita <= 0) {
+    $volnaKapacita = "Kapacita je vyčerpána";
+}
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Party <?php echo $nazevParty; ?></title>
+</head>
+<body>
+    <h1>Party <?php echo $nazevParty; ?></h1>
+
+    <ul>
+        <li>Pořadatel: <?php echo $poradatel; ?></li>
+        <li>Místo: <?php echo $misto; ?></li>
+        <li>Počet míst: <?php echo $kapacita; ?></li>
+        <li>Zbývá míst: <?php echo $volnaKapacita; ?></li>
+        <li>Příjde: <?php echo $pocetUcastniku; ?></li>
+    </ul>
+
+    <h2>Účastníci</h2>
+    <ol>
+        <?php
+        foreach ($ucastnici as $ucastnik) {
```

Comment:
Nechceme to udělat celý v PHP?
```php
foreach ($ucastnici as $ucastnik) {
    echo "<li>";
    echo $ucastnik;
    echo "</li>";
}
```
?
Jinde v ridicich strukturach tohle prechazeni mezi HTML a PHP nemame, tak jestli se tomu raději nevyhnout.

Reply:
foreach je tvuj.. tak si to klidne zmen